giskard_vision/scanner/scanner.py

```diff
@@ -5,8 +5,13 @@
 from time import perf_counter
 from typing import Any, Optional, Sequence
 
-from giskard.scanner.registry import DetectorRegistry
-from giskard.scanner.report import ScanReport
+from ..utils.errors import GiskardImportError
+
+try:
+    from giskard.scanner.registry import DetectorRegistry
+    from giskard.scanner.report import ScanReport
+except (ImportError, ModuleNotFoundError) as e:
+    raise GiskardImportError(["giskard"]) from e
```

Comment:
we don't want the library to break if we ever do `from giskard_vision.scanner import Scanner` even if `giskard` is not installed.

For that reason I asked to embed these try except clauses inside the methods of the class.